cms/models.py

```diff
@@ -46,9 +46,9 @@ def get_context(self, request):
         context["programs"] = Program.objects.filter(live=True)
         context["style_src"] = get_bundle_url(request, "style.js")
         context["public_src"] = get_bundle_url(request, "public.js")
-        context["custom_src"] = get_bundle_url(request, "js/home_page/custom.js")
-        context["ajaxchimp_src"] = get_bundle_url(request, "js/home_page/jquery.ajaxchimp.min.js")
-        context["wow_src"] = get_bundle_url(request, "js/home_page/wow.min.js")
+        context["custom_src"] = static("js/home_page/custom.js")
+        context["ajaxchimp_src"] = static("js/home_page/jquery.ajaxchimp.min.js")
+        context["wow_src"] = static("js/home_page/wow.min.js")
```

Comment:
custom.js was added in PR #306, copied from here: https://github.com/mitodl/mga/blob/gh-pages/js/custom.js

I think we should roll `custom.js` into the public bundle and leave the other two files as static assets. I'm not sure if the mailchimp code would be in npm to import, and in any case we don't manage jquery with npm, it's included in `base.html` from a CDN